homeassistant/components/ecobee/manifest.json

```diff
@@ -10,7 +10,7 @@
   },
   "iot_class": "cloud_polling",
   "loggers": ["pyecobee"],
-  "requirements": ["python-ecobee-api==0.2.17"],
+  "requirements": ["python-ecobee-api==0.2.18"],
```

Comment:
This is to get access to the location data, this change IS related. it is not just a version bump